unity-renderer/Assets/Scripts/MainScripts/DCL/Components/Video/Plugins/WebVideoPlayer.jslib

```diff
@@ -142,11 +142,11 @@ var WebVideoPlayer = {
         GLctx.texImage2D(
             GLctx.TEXTURE_2D,
             0,
-            GLctx.SRGB8_ALPHA8,
+            GLctx.RGBA8,
```

Comment:
If this really fixes the issue, please let's add a contundent comment and warning for future developers 🙏

Reply:
good point, will do it!

Reply:
bad news, this fix is removing color space from the equation, making video to be brighter :( I am continue to investigate in another possible solution